server.py

```diff
@@ -53,6 +53,7 @@
         "134": ["0005"],
         "139": ["0001"],
         "144": ["0001"],
+        "019": ["0018", "0019", "0020"]
```

Comment:
Super picky but can you put it in number order

Reply:
@insacuri Done